pulpcore/cli/common/generic.py

```diff
@@ -742,7 +742,7 @@ def callback(
         """
         Show the list of optionally filtered {entities}.
         """
-        parameters = {k: v for k, v in kwargs.items() if v is not None}
+        parameters = entity_ctx.preprocess_body(kwargs)
```

Comment:
Are you sure, this is what we want here? I expect the parameters to be filters mostly (exclusively?) in case of list commands.

Reply:
I just wanted to use a `resource_option` as a filter option, and `preprocess_body` calls `_preprocess_value` which will convert a context to a href in the body. Maybe I should switch the call since `preprocess_body` can change arguments (usually for create calls) which might not be ideal.